.github/workflows/release_pull_request.yml

```diff
@@ -44,16 +44,16 @@ jobs:
           git config --global user.name 'VA Automation Bot'
           git config --global user.email 'va-mobileapp@adhocteam.us'
           git checkout release/${{inputs.version}}
-          git checkout -b dev-to-release
-          git push -u origin dev-to-release
+          git checkout -b release-${{inputs.version}}-to-develop
```

Comment:
Since this branch name is used more than once, consider defining a variable above, i.e.

`MERGE_TO_DEV_BRANCH="release-${{inputs.version}}-to-develop"`

Then use it on lines 47 and 48 in the `git` commands. This will allow us to only change it in one spot if we need to in the future, and avoid any potential typos from repeating the expression.

The same goes for `merge-${{inputs.version}}-to-main`.